### DISCUSSION TITLE
XTOTV UPDATE

### DISCUSSION THREAD

**Discussion by LEODJ69** [iptv-org/community#1105] [Q&A]
Could you please update the data of XTOTV Bolivian channel?
web: https://www.sccbolivia.com/
live stream: http://190.104.15.135/0.ts
logo: https://www.sccbolivia.com/wp-content/uploads/2020/09/xtotv-tx-inicio.png

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/bo.m3u#L138-L140
